modules/social_features/social_follow_taxonomy/modules/social_follow_tag/social_follow_tag.info.yml

```diff
@@ -0,0 +1,8 @@
+name: 'Social Follow Tag'
+description: 'Provide follow functionality for social_tagging module.'
+type: module
+core: 8.x
+dependencies:
+  - social_follow_taxonomy:social_follow_taxonomy
+  - social_tagging:social_tagging
+package: Social
```

Comment:
@denysyukm I think we need to add it to the experimental package as well.

Reply:
DONE